README.md

```diff
@@ -119,22 +119,22 @@ v run tetris/tetris.v
 
 <img src='https://raw.githubusercontent.com/vlang/v/master/examples/tetris/screenshot.png' width=300>
 
-In order to build Tetris and anything else using the graphics module, you will need to install glfw and freetype libraries.
+To build Tetris and anything else using the graphics module, you will need to install glfw and FreeType libraries.
 
-If you plan to use the http package, you also need to install OpenSSL on non-Windows systems.
+If you plan to use the Http package, you also need to install OpenSSL on non-Windows systems.
 
 ```
 macOS:
-brew install glfw freetype openssl
+brew install glfw FreeType OpenSSL
 
 Debian/Ubuntu:
 sudo apt install libglfw3 libglfw3-dev libfreetype6-dev libssl-dev
 
 Arch/Manjaro:
-sudo pacman -S glfw-x11 freetype2
+sudo Pacman -S glfw-x11 freetype2
```

Comment:
Why is `pacman` capitalized?

Reply:
not just that, there are several unnecessary capitalization in this PR